### DISCUSSION TITLE
what is means to this code? (in html)

### DISCUSSION THREAD

**Discussion by codebimbambum** [community/community#46843] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

this code is opensource. i know code 'link'.
but, into 'href' i don't understand to text. 
so help. what is means to this code? 

<link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"
      integrity="sha384-v8BU367qNbs/aIZIxuivaU55N5GPF89WBerHoGA4QTcbUjYiLQtKdrfXnqAcXyTv"
      crossorigin="anonymous"
    />

**Comment by codebimbambum** [comment]
`<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css" integrity="sha384-v8BU367qNbs/aIZIxuivaU55N5GPF89WBerHoGA4QTcbUjYiLQtKdrfXnqAcXyTv" crossorigin="anonymous" />`

**Reply by jge162** [reply]
The code you provided is a link to a stylesheet that provides access to Font Awesome icons.

**Reply by ghost** [reply]
You're right

**Comment by SilasRodrigues19** [comment]
Hey @kirokrobot 

`href` is an HTML property that specifies the address of the style file that will be used to format the page. In the case of your example, this attribute specifies the URL of the [Font Awesome](https://fontawesome.com/) CSS library file.

`integrity` is an additional property that provides the browser with a way to check if the content of the file has been altered since it was loaded.
